frontend/src/components/Reports/Routine.js

```diff
@@ -58,21 +55,15 @@ export const RoutineReportsMenu = {
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=activityReportByTest",
+          link: "/ActivityReport?type=indicator&report=activityReportByTest",
```

Comment:
i dont see an Route handling the path `ActivityReport `. 
can you remove this change ??